src/Components/CartCard/CartCard.js

```diff
@@ -25,24 +29,44 @@ export default function CartCard() {
                 <div className="description1">
                   <p className="item-price">₹ {product.price}</p>
                   <p>
-                    <label htmlFor="quantity">Quantity:</label>
-                    <input
-                      className="itemQuantity"
-                      type="number"
-                      id="quantity"
-                      name="quantity"
-                      min="1"
-                      max="5"
-                      value={product.quantity}
-                    />
+                    <label htmlFor="quantity">
+                      Quantity:{" "}
+                      <button
+                        className="btn-changeQty"
+                        onClick={() =>
+                          dispatch({
+                            type: "REDUCE",
+                            payload: { product: product },
+                          })
+                        }
+                      >
+                        {product.quantity === 1 ? (
+                          <i class="fas fa-trash"></i>
+                        ) : (
+                          "-"
+                        )}
+                      </button>
+                      {product.quantity}
+                      <button
+                        className="btn-changeQty"
+                        onClick={() =>
+                          dispatch({
+                            type: "ADDED",
```

Comment:
this ADDED can be move to file called actionTypes and used at any place we want

Reply:
That seems like a good idea.